docs/userguide/objective.md

```diff
@@ -1,24 +1,77 @@
 # Objective
 
-## General Information
+The [`Objective`](baybe.objective.Objective) instructs BayBE how to deal with multiple
+targets (if applicable).
 
-Objectives allows you to effectively manage the optimization objective and are implemented by the [`Objective`](baybe.objective.Objective) class.
-By defining objectives, you can optimize a single target function or combine multiple target functions using different modes.
-To create an objective, it is necessary to provide at least the following:
-* **An optimization mode:** This can be either ``SINGLE`` or ``DESIRABILITY``, denoting either the optimization of a single target function or a combination of different target functions.
-* **A list of targets:** The list of targets that are optimized. Note that the ``SINGLE`` mode also requires a list containing the single target. See [here](./targets) for details.
+To create an objective, it is necessary to provide the following:
+* An optimization `mode`: This can be either `SINGLE` or `DESIRABILITY`,
+  denoting the optimization of a single target function or a combination of
+  different target functions respectively.
+* A list of `targets`: The list of targets that are optimized. Note that the
```

Comment:
Just as an idea: Do we want to have a link to either the target class here or the corresponding user guide? Not strictly necessary imo, so if you feel that this is not necessary just mark as resolved.

Reply:
agree, icluded